erp/schema.py

```diff
@@ -67,6 +67,17 @@
     (HANDICAP_VISUEL, "Visuel"),
 ]
 
+DISPOSITIFS_APPEL_BOUTON = "bouton"
+DISPOSITIFS_APPEL_SONNETTE = "sonnette"
+DISPOSITIFS_APPEL_INTERPHONE = "interphone"
+DISPOSITIFS_APPEL_VISIOPHONE = "visiophone"
+ENTREE_DISPOSITIFS_APPEL_CHOICES = [
+    (DISPOSITIFS_APPEL_BOUTON, "Bouton d’appel"),
+    (DISPOSITIFS_APPEL_SONNETTE, "Sonnette"),
+    (DISPOSITIFS_APPEL_INTERPHONE, "Interphone"),
+    (DISPOSITIFS_APPEL_VISIOPHONE, "Visiophone")
```

Comment:
Is the `ENTREE_` prefix necessary? If so, why it's not used for every *constants*? I'd vote for ditching the prefix.

Reply:
Gotcha, i wanted to keep for consistency, but shortened the internals for sake of brevity.